src/arrays.jl

```diff
@@ -24,6 +24,9 @@ function deepdiff(X::Vector, Y::Vector)
 
     lengths = zeros(Int, length(X)+1, length(Y)+1)
     backtracks = fill(:nothing, axes(lengths))
+    backtracks[1,2:end] .= :X
+    backtracks[2:end,1] .= :Y
+    backtracks[1,1] = :nothing
```

Comment:
Unnecessary

Reply:
This is an "intermediate" commit (these lines are not present in the PR if we decide to sqash the commits), and in any case it's necessary in that commit if I'm not mistaken. Let me know if I misunderstood, or am mistaken.